pallets/murmur/src/mock.rs

```diff
@@ -319,16 +285,24 @@ pub fn new_test_ext_raw_authorities(authorities: Vec<(u64, BeefyId)>) -> TestExt
 		}
 	});
 
-	let (round_pubkey, genesis_resharing) = etf_genesis::<w3f_bls::TinyBLS377>(
-		authorities.iter().map(|(_, id)| id.clone()).collect::<Vec<_>>(),
-	);
-
-	pallet_etf::GenesisConfig::<Test> {
-		genesis_resharing,
-		round_pubkey,
-	}
-	.assimilate_storage(&mut t)
-	.unwrap();
+	// mock the genesis config
+	let genesis_resharing: Vec<(sp_consensus_beefy_etf::bls_crypto::Public, Vec<u8>)> =
+		vec![(mock_beefy_id(123), [1u8; 32].into())];
+	let round_pubkey = [
+		144, 122, 123, 77, 192, 77, 117, 246, 132, 139, 163, 31, 26, 99, 75, 76, 23, 206, 24, 252,
+		200, 112, 18, 199, 82, 203, 96, 23, 70, 76, 156, 253, 67, 126, 106, 164, 154, 25, 154, 95,
+		155, 32, 173, 48, 126, 0, 123, 129, 86, 203, 71, 65, 207, 131, 55, 168, 72, 235, 88, 180,
+		5, 20, 167, 118, 31, 36, 35, 125, 250, 33, 33, 224, 230, 106, 155, 79, 79, 137, 130, 57,
+		146, 66, 236, 129, 17, 178, 199, 180, 48, 108, 247, 161, 0, 139, 7, 0, 180, 41, 114, 7, 69,
+		134, 33, 178, 54, 23, 119, 67, 67, 173, 76, 36, 94, 29, 1, 134, 114, 228, 28, 69, 152, 14,
+		57, 17, 38, 6, 83, 43, 155, 211, 188, 64, 91, 193, 205, 125, 222, 52, 19, 237, 173, 184,
+		129, 128,
+	]
+	.into();
+
```

Comment:
@driemworks not sure if you saw this in within all the changes, but this is mainly the murmur test fix

Reply:
I might replace it with a 'real' etf_genesis setup in the future but I'm happy with the change